src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -169,6 +184,10 @@ UniversalNavbarExpanded.propTypes = {
             title: PropTypes.string,
             id: PropTypes.string,
             subcopy: PropTypes.string,
+            alternateIcon: PropTypes.oneOfType([
+              PropTypes.element,
+              PropTypes.bool,
+            ]),
```

Comment:
Because we're using propTypes and not typescript, it's hard to make `alternateIcon` optional while still requiring a shape for the navlinks object, so I went with the option to pass a boolean or an element.